modules/zip/src/tar-builder.ts

```diff
@@ -1,14 +1,16 @@
 import Tar from './lib/tar/tar';
 
+type TAR_BUILDER_OPTIONS = {
```

Comment:
We use `PascalCase` for types, and `CAPITAL_CASE` for constants:
```suggestion
type TarBuilderOptions = {
```